steps/recursive-exception-reports.sh

```diff
@@ -17,28 +17,22 @@ else
 fi
 
 echo -n "[*] Populating staging table and exporting CSV... "
-#psql -h "$DATA_STORE_HOST" -p "$DATA_STORE_PORT" -d "$DATA_STORE_NAME" -U "$DATA_STORE_USER" -w -v FROM_DATE="'${FROM_DATE}'" -v JURISDICTION="'${JURISDICTION}'" -f scripts/migrate-staging.sql data_store_snapshot 2>&1 > /dev/null
 
 if [ $OPERATION$DBTYPE = "exportrecursiveexceptionrealtime" ]; then
     echo "EXPORTING Recursive DOCUMENT IDs : Exporting Document Ids from Temp DB $DATA_STORE_HOST  $DATA_STORE_PORT $DATA_STORE_NAME $DATA_STORE_USER"
     export PGPASSWORD="$DATA_STORE_PASS"
-    psql sslmode=true -h "$DATA_STORE_HOST" -p "$DATA_STORE_PORT" -d "$DATA_STORE_NAME" -U "$DATA_STORE_USER" -W  -f scripts/recursive-exception-report.sql
+    psql -h "$DATA_STORE_HOST" -p "$DATA_STORE_PORT" -d "$DATA_STORE_NAME" -U "$DATA_STORE_USER" -f scripts/recursive-exception-report.sql
```

Comment:
sslmode still required for non-local DBs

Reply:
Fixed in 9e467861f2e8d3c0d88b9eee9441cd3269f7fb69